frontend/src/hocs/withScreencast/withScreencast.tsx

```diff
@@ -77,51 +69,49 @@ function withScreencast(
       currentState: "OFF" as Steps,
     }
 
-    toggleRecordAudio = (): void => {
+    private toggleRecordAudio = (): void => {
       const { recordAudio } = this.state
 
       this.setState({
         recordAudio: !recordAudio,
       })
     }
 
-    showDialog = (fileName: string): void => {
+    private showDialog = (fileName: string): void => {
       const { currentState } = this.state
 
-      if (!this.checkSupportedBrowser()) {
-        this.setState({
-          currentState: "UNSUPPORTED",
-        })
+      if (!ScreenCastRecorder.isSupportedBrowser()) {
+        this.setState({ currentState: "UNSUPPORTED" })
       } else if (currentState === "OFF") {
         this.setState({
           fileName,
           currentState: "SETUP",
         })
       } else {
-        this.stopRecording()
+        this.stopRecording().catch(err =>
+          logWarning(`withScreencast.stopRecording threw an error: ${err}`)
+        )
       }
     }
 
-    startRecording = async (): Promise<any> => {
+    public startRecording = async (): Promise<any> => {
       const { recordAudio } = this.state
 
       this.recorder = new ScreenCastRecorder({ recordAudio })
 
       try {
         await this.recorder.initialize()
       } catch (e) {
-        // Just in case if there's something wrong when initializing the recorder
-        this.setState({
-          currentState: "UNSUPPORTED",
-        })
+        // There was something wrong initializing the recorder.
```

Comment:
Nit: There was something wrong **with**(?) initializing the recorder.

Reply:
I'm just going to remove the comment altogether - it's not adding anything here